bc_obps/registration/management/commands/create_superuser.py

```diff
@@ -1,19 +1,20 @@
 import os
+
 from django.conf import settings
-from django.core.management.base import BaseCommand
 from django.contrib.auth import get_user_model
 from django.core.management import call_command
+from django.core.management.base import BaseCommand
 
 
 class Command(BaseCommand):
     help = 'Create a superuser with username and password'
 
     def handle(self, *args, **options):
         User = get_user_model()
-        if settings.DEBUG:
-            superuser_username = os.environ.get('SUPERUSER_USERNAME')
-            superuser_password = os.environ.get('SUPERUSER_PASSWORD')
-            if not superuser_username or not superuser_password:
+        superuser_username = os.environ.get('DJANGO_SUPERUSER_USERNAME')
+        superuser_password = os.environ.get('DJANGO_SUPERUSER_PASSWORD')
+        if superuser_username and superuser_password:
+            User.objects.create_superuser(username=superuser_username, password=superuser_password)
+        else:
+            if settings.DEBUG:  # we don't want to call this in production
```

Comment:
just want to make sure I'm understanding this correctly - the way we ensure that the Django admin panel isn't available in prod is that we won't be setting env vars in prod for DJANGO_SUPERUSER_x and DEBUG will be turned off?

Reply:
Yes, and also we only run `create_superuser` command in dev.